similarity-data-impl/src/main/java/org/phenotips/data/similarity/internal/DefaultPatientGenotype.java

```diff
@@ -48,9 +49,14 @@
  */
 public class DefaultPatientGenotype extends AbstractExome implements PatientGenotype
 {
+
     /** Factory for loading exome data. */
     protected static ExomeManager exomeManager;
 
+    /** List of genes variant interpretations of pathogenic status 3-5 (VUS or higher). */
+    private static final String[] TOP_INTERPRETATIONS =
+    {"Pathogenic", "Likely Pathogenic", "Variant of Unknown Significance"};
```

Comment:
getData("variants") returns already translated values, as I understand, getData() calls all controllers lo load() data, which comes translated for external use/display/export